### PR TITLE
Dev v0.6.0 pkg refactor

### PR DESCRIPTION
**Pull request by jwildfire**
Updates to CreatePackage and associated files.

### CODE REVIEW

File: R/CreatePackage.R [hunk 1]
```diff
@@ -1,58 +1,26 @@
...
+    opts <- list()
+    if(strAuthors != "") opts[`Authors@R`]=strAuthors
+    usethis::create_package(path=strPackagePath, fields=opts, check_name=FALSE, open=FALSE)
+    usethis::with_project(path=strPackagePath,usethis::use_testthat())
+
+    if(bSampleFiles){
+        strSrcDir<-paste0(GetTemplateDirectory(),"/Pkg")
+        strDestDir<- paste0(strPackagePath)
+        CopyFiles(strSrcDir, strDestDir)
     }
```
Comment (nchen4) [inline R/CreatePackage.R:25]: It seems the AUTHOR_NAME tag is not created in the file, so the future rendering for that part is missing?

File: inst/ProjectTemplate/Pkg/README.md [hunk 2]
```diff
@@ -1,9 +1,4 @@
----
-title: "Instructions"
-author: "{{AUTHOR_NAME}}"
-date: "8/10/2020"
-output: html_document
----
+
```
Comment (nchen4) [inline inst/ProjectTemplate/Pkg/README.md:1]: Why this section is removed?

Comment (jwildfire) [reply inst/ProjectTemplate/Pkg/README.md:1]: I converted from RMD to standard markdown here, so you don't generally use the header in .md files I've seen.

File: R/CreatePackage.R (hunk 1 above)
Comment (jwildfire) [reply R/CreatePackage.R:25]: Right - I've converted this to use the `fields` option in usethis::create_package to fill in the description file instead.

**Pull request merged**